### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by patryk-babon**
- [DEMO LINK](https://patryk-babon.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://patryk-babon.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,50 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
```
Comment (choeqq) [inline src/index.html:15]: good `<a />` tag wrapping 👍

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,50 @@
...
+            <a class="nav__link" href="#">GADGETS</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">TABLETS</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">PHOTO</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">VIDEO</a>
```
Comment (choeqq) [inline src/index.html:50]: in requirements we said that we want to uppercase list item with css property

**Review by choeqq — CHANGES_REQUESTED** [review]

**Review by asakevych — APPROVED** [review]
Great job!! 🚀 👏 👏

**Review by choeqq — APPROVED** [review]
All good 🚀

**Review by mvjl000 — APPROVED** [review]

**Pull request closed without merging**